src/infra/dex/balancer/mod.rs

```diff
@@ -80,6 +81,8 @@ impl Sor {
             self.chain_id,
             self.settlement,
             self.query_batch_swap,
+            // 2 minutes from now
+            (chrono::Utc::now().timestamp() + 120).to_u64(),
```

Comment:
What about summing up the UTC time in seconds and then transforming it to timestamp? :thinking: it is clearer we are adding two minutes